<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FilterForm</class>
 <widget class="QWidget" name="FilterForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>646</width>
    <height>102</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>7</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Sise&lt;=(MB)</string>
       </property>
      </widget>
     </item>
     <item row="2" column="5">
      <widget class="ComboEdit" name="name3Edit"/>
     </item>
     <item row="0" column="4">
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <pointsize>7</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item row="0" column="7" rowspan="3">
      <widget class="QGroupBox" name="showGroupBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>79</width>
         <height>0</height>
        </size>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QCheckBox" name="showExistingCheckBox">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>60</width>
          <height>22</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>7</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Exist</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QCheckBox" name="showDeletedCheckBox">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>35</y>
          <width>60</width>
          <height>22</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>7</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Del</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QCheckBox" name="showPartCheckBox">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>17</y>
          <width>60</width>
          <height>22</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>7</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Part</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QCheckBox" name="birthCheckBox">
        <property name="geometry">
         <rect>
          <x>2</x>
          <y>59</y>
          <width>60</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Birth</string>
        </property>
       </widget>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="ComboEdit" name="name2Edit">
       <property name="font">
        <font>
         <pointsize>7</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <layout class="QVBoxLayout" name="verticalLayout"/>
     </item>
     <item row="1" column="8">
      <widget class="QPushButton" name="appendButton">
       <property name="font">
        <font>
         <pointsize>7</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Append</string>
       </property>
      </widget>
     </item>
     <item row="0" column="8">
      <widget class="QPushButton" name="okButton">
       <property name="font">
        <font>
         <pointsize>7</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Search</string>
       </property>
      </widget>
     </item>
     <item row="0" column="6" rowspan="2">
      <widget class="QGroupBox" name="sizeGroupBox">
       <property name="title">
        <string/>
       </property>
       <widget class="QRadioButton" name="bytesRadioButton">
        <property name="geometry">
         <rect>
          <x>5</x>
          <y>4</y>
          <width>50</width>
          <height>22</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>7</pointsize>
         </font>
        </property>
        <property name="text">
         <string>B</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="mbytesRadioButton">
        <property name="geometry">
         <rect>
          <x>5</x>
          <y>23</y>
          <width>50</width>
          <height>22</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>7</pointsize>
         </font>
        </property>
        <property name="text">
         <string>MB</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
     </item>
     <item row="2" column="8">
      <widget class="QPushButton" name="discardButton">
       <property name="font">
        <font>
         <pointsize>7</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Discard</string>
       </property>
      </widget>
     </item>
     <item row="2" column="6">
      <widget class="QPushButton" name="unsortButton">
       <property name="font">
        <font>
         <pointsize>7</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Unsort</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <pointsize>7</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Size&gt;=(MB)</string>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QPushButton" name="clearButton">
       <property name="palette">
        <palette>
         <active>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="ButtonText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="ButtonText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>120</red>
             <green>120</green>
             <blue>120</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="ButtonText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>120</red>
             <green>120</green>
             <blue>120</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
       <property name="font">
        <font>
         <pointsize>7</pointsize>
        </font>
       </property>
       <property name="text">
        <string>X</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QSpinBox" name="sizeMoreSpinBox">
       <property name="accelerated">
        <bool>true</bool>
       </property>
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>60000</number>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="ComboEdit" name="name1Edit">
       <property name="font">
        <font>
         <pointsize>7</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <pointsize>7</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Size=(B)</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QSpinBox" name="sizeLessSpinBox">
       <property name="maximum">
        <number>60000</number>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QLineEdit" name="sizeEqEdit">
       <property name="font">
        <font>
         <pointsize>7</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="clearMBytesButton">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>20</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="palette">
        <palette>
         <active>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="ButtonText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="ButtonText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>120</red>
             <green>120</green>
             <blue>120</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="ButtonText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>120</red>
             <green>120</green>
             <blue>120</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
       <property name="text">
        <string>x</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ComboEdit</class>
   <extends>QLineEdit</extends>
   <header>comboedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
